test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -1229,6 +1229,12 @@ public void ExcludedFunction_NotAddedToHost()
             Assert.Equal(1, array[1]);
         }
 
+        public void Dispose()
+        {
+            // Make sure all logs are flushed before next test.
```

Comment:
Yeah, that's what I'm realizing! The problem is the "Scenarios" test where the test before this ("NextTick") writes a couple logs and they aren't flushed when I delete the file -- so they occasionally pop in. My only solution here may be to either write the test code to work either way -- or wait 1 second (the file logger flush interval) to make sure they're flushed before I delete the file. 

I'll try the delay first and run this PR a few times to see if it works.